src/components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  filter: string;
+  onSetFilter: (filter: string) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  onSetFilter,
+}) => (
+  <footer className="todoapp__footer">
+    <span className="todo-count">
+      {`${todos.filter(todo => !todo.completed).length} items left`}
```

Comment:
it would be better to create `activeTodos` variable before rendering
`const activeTodos = todos.filter(todo => !todo.completed).length;`